data/json/itemgroups/magazines.json

```diff
@@ -141,6 +144,7 @@
       ["fnp90mag", 15],
       ["mp5bigmag", 10],
       ["thompson_bigmag", 5],
+      ["ppsh35mag", 5]
```

Comment:
The other SMG magazines belong here, again note alphabetical ordering - this is required to limit merge conflicts